tardis/io/schemas/csvy_model.yml

```diff
@@ -0,0 +1,147 @@
+$schema: http://json-schema.org/draft-04/schema#
+type: object
+additionalProperties: false
+properties:
+  name: 
+    type: string
+    description: name of the model being run
+
+  description:
+    type: string
+    description: description of the model being run
+
+  tardis_model_config_version:
+    type: string
+    description: Version of the configuration file
+
+  atom_data:
+    type: string
+    description: path to the atom data
+
+  datatype:
```

Comment:
why do we call it 'dataype' - please add comment

Reply:
this is not required

Reply:
Added comment: "datatype name used for consistency with astropy ecsv files."